code/modules/jobs/job_types/head_of_personnel.dm

```diff
@@ -55,6 +60,14 @@
 	if(locate(/datum/holiday/ianbirthday) in SSevents.holidays)
 		undershirt = /datum/sprite_accessory/undershirt/ian
 
+//only pet worth reviving
+/datum/job/hop/get_mail_goodies(mob/recipient)
+	. = ..()
+	// Strange Reagent if the pet is dead.
+	for(var/mob/living/simple_animal/pet/dog/corgi/ian/staff_pet in GLOB.dead_mob_list)
+		. += list(/datum/reagent/medicine/strange_reagent = 20)
```

Comment:
you might wanna use a lazarus injector instead here; it'd be more intuitive for newer HoPs to use

Reply:
I'd also recommend either cranking the odds of this being rolled way up or moving this to the normal HoP letter items list